packages/edit-site/src/components/sidebar-navigation-screen-templates/index.js

```diff
@@ -64,11 +71,14 @@ export default function SidebarNavigationScreenTemplates() {
 		path: '/' + postType + '/all',
 	} );
 
+	const canCreate = ! isMobileViewport && ! isTemplatePartsMode;
+
 	return (
 		<SidebarNavigationScreen
+			isRoot={ isTemplatePartsMode }
```

Comment:
This is a bit weird to be honest, I think the ideal solution for this "mode" is to probably have its own "init function" rather than rely on the default site editor initialization function. In the long term, I'm certain this "hack" is going to bite us. By "Hack" I mean the fact that we kind of try to load a specific version of the site editor for the "template parts" screen of classic themes.

Reply:
It's still the same site editor; we don't want some routes to be available for Classic themes.

But I agree this "hack" isn't the best way to solve this problem. I will try to explore different init methods for WP 6.3.